lib/model/store.dart

```diff
@@ -470,6 +474,7 @@ class PerAccountStore extends ChangeNotifier with StreamStore, MessageStore {
       assert(debugLog("server event: message ${jsonEncode(event.message.toJson())}"));
       _messages.handleMessageEvent(event);
       unreads.handleMessageEvent(event);
+      recentSenders.handleMessage(event.message);
       recentDmConversationsView.handleMessageEvent(event);
```

Comment:
nit: put these calls in the same order the fields are declared

(unless there's some dependency between them that forces an ordering — but if there were then that should be explicitly commented)